src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/appsettings.json

```diff
@@ -10,5 +10,5 @@
   },
   "UseStubProviderRelationships": false,
   "UseStubProviderAuth": false,
-  "UseLocalRegistry": false
+  "UseLocalRegistry": true
```

Comment:
should we be checking in changes to these settings?

Reply:
Some one else checked in true few weeks ago, just reverting it back to the way it was before.

Reply:
This shouldn't go in, will make the change